AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/scripting_tools_qt.py

```diff
@@ -0,0 +1,556 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from editor_python_test_tools.utils import TestHelper as helper
+from PySide2 import QtWidgets, QtTest, QtCore
+from PySide2.QtCore import Qt
+from editor_python_test_tools.utils import Report
+import pyside_utils
+import editor_python_test_tools.hydra_editor_utils as hydra
+from editor_python_test_tools.editor_entity_utils import EditorEntity
+import editor_python_test_tools.EditorQtContainer as qtContainer
+import azlmbr.editor as editor
+import azlmbr.math as math
+import azlmbr.bus as bus
+import azlmbr.legacy.general as general
+import azlmbr.scriptcanvas as scriptcanvas
+from scripting_utils.scripting_constants import (SCRIPT_CANVAS_UI, ASSET_EDITOR_UI, NODE_PALETTE_UI, NODE_PALETTE_QT,
+                                                 TREE_VIEW_QT, SEARCH_FRAME_QT, SEARCH_FILTER_QT, SAVE_STRING, NAME_STRING,
+                                                 SAVE_ASSET_AS, WAIT_TIME_3, NODE_INSPECTOR_TITLE_KEY, WAIT_FRAMES,
+                                                 VARIABLE_MANAGER_QT, NODE_INSPECTOR_QT, NODE_INSPECTOR_UI, SCRIPT_EVENT_UI,
+                                                 VARIABLE_PALETTE_QT, ADD_BUTTON_QT, VARIABLE_TYPES, EVENTS_QT, DEFAULT_SCRIPT_EVENT,
+                                                 SCRIPT_EVENT_FILE_PATH, PARAMETERS_QT, VARIABLE_MANAGER_QT, NODE_INSPECTOR_QT,
+                                                 NODE_INSPECTOR_UI, VARIABLE_PALETTE_QT, ADD_BUTTON_QT, VARIABLE_TYPES,
+                                                 SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, ENTITY_STATES)
+
+
+class Tests():
+    new_event_created = ("New Script Event created", "Failed to create a new event")
+    child_event_created = ("Child Event created", "Failed to create Child Event")
+    parameter_created = ("Successfully added parameter", "Failed to add parameter")
+    parameter_removed = ("Successfully removed parameter", "Failed to remove parameter")
+
+def click_menu_option(window, option_text):
```

Comment:
The more I look at this, the more I feel like this should be a class that initializes itself and setup. Some of the behaviors I've suggested to change in the test file would be able to be cleaned up by using a class, but is not the only way.

Reply:
added 2 new wrapper classes for the qt objects and placed scripting tools behavior into them